textarea_select.html

```diff
@@ -71,6 +71,19 @@ <h1>Form Autofill Demo Page</h1>
           <option value="WV">West Virginia</option>
           <option value="WI">Wisconsin</option>
           <option value="WY">Wyoming</option>
+          <option value="AB">Alberta</option>
```

Comment:
Nit: It might be nice to wrap both countries in an `<optgroup>`

Reply:
Thanks!